MeetupPlatformApi/DataTransferObjects/MeetupOutputDto.cs

```diff
@@ -1,14 +1,23 @@
-﻿namespace MeetupPlatformApi.DataTransferObjects;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.DataTransferObjects;
 
 public class MeetupOutputDto
 {
+    [Required]
     public int Id { get; set; }
 
+    [Required]
+    [MaxLength(30)]
```

Comment:
I'm not sure if we need `[MaxLength]` attributes on output models. If You insist, we can keep them, but I personally don't see any real purpose in using them (especially considering that we might decide to change max-length for some field: that means, that we will validate by new limit, but will also retrun previously added values that we have in our database).